commit 0aeefcc4f7a5fa843d99bbaec2ff71be7d722c9a
Author: Pavel Labath <labath@google.com>
Date:   Thu Mar 2 11:36:14 2017 +0000

    Fix flakyness in TestGdbRemoteHostInfo
    
    this test was using the VPATH hack to avoid having a copy of the
    inferior source code. This makes the test fail if in happens to run
    concurrently with a test in the parent folder. Fix that by moving it up
    to the parent.
    
    git-svn-id: https://llvm.org/svn/llvm-project/lldb/trunk@296741 91177308-0d34-0410-b5e6-96231b3b80d8

diff --git a/packages/Python/lldbsuite/test/tools/lldb-server/host-info/TestGdbRemoteHostInfo.py b/packages/Python/lldbsuite/test/tools/lldb-server/TestGdbRemoteHostInfo.py
similarity index 100%
rename from packages/Python/lldbsuite/test/tools/lldb-server/host-info/TestGdbRemoteHostInfo.py
rename to packages/Python/lldbsuite/test/tools/lldb-server/TestGdbRemoteHostInfo.py
diff --git a/packages/Python/lldbsuite/test/tools/lldb-server/host-info/Makefile b/packages/Python/lldbsuite/test/tools/lldb-server/host-info/Makefile
deleted file mode 100644
index 1370b53..0000000
--- a/packages/Python/lldbsuite/test/tools/lldb-server/host-info/Makefile
+++ /dev/null
@@ -1,10 +0,0 @@
-LEVEL = ../../../make
-
-VPATH = ..
-
-override CFLAGS_EXTRAS += -D__STDC_LIMIT_MACROS -D__STDC_FORMAT_MACROS
-ENABLE_THREADS := YES
-CXX_SOURCES := main.cpp
-MAKE_DSYM :=NO
-
-include $(LEVEL)/Makefile.rules
